ci/requirements-py27-pydap.yml

```diff
@@ -8,6 +10,7 @@ dependencies:
   - numpy
   - pandas
   - scipy
+  - rasterio
```

Comment:
I don't think we need to test rasterio along with pydap so we can roll back these changes.